### PR TITLE
Document Memcached cache adapter added in Symfony 3.3

### LINKED ISSUE #7828: [Cache] Documentation for default_memcached_provider
A new `default_memcached_provider` option was introduced in symfony/symfony#21108.

### PR DESCRIPTION
**Pull request by aik099**
Adds documentation for Memcached cache adapter.

Related blog post: https://symfony.com/blog/new-in-symfony-3-3-memcached-cache-adapter

Closes #7828

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1776,6 +1777,18 @@ default_redis_provider
 The DSN to use by the Redis provider. The provider is available as the ``cache.redis``
 service.
 
+default_memcached_provider
+..........................
+
+.. versionadded:: 3.3
+
```
Comment (wouterj) [inline reference/configuration/framework.rst]: This empty line should be removed (versionadded is kinda a strange directive that requires this...)

**Review by wouterj — APPROVED** [review]
Thanks! I've left one minor comment, but that can be fixed during the merge

File: reference/configuration/framework.rst (hunk 1 above)
Comment (aik099) [reply reference/configuration/framework.rst]: Done.

**Comment by aik099** [comment]
I wonder why the Platform.sh build was triggered on my other PR, but not on this one.

**Comment by wouterj** [comment]
It is triggered (at least now) if you collapse the status item you can see it.

**Comment by xabbuh** [comment]
Thank you @aik099.

**Pull request merged**